package.json

```diff
@@ -6,7 +6,8 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "toggle-local-stytch": "(npm ls stytch | grep ./../stytch-node && npm uninstall stytch && npm install stytch) || (npm ls stytch | grep -q ./../stytch-node || npm install --save ../stytch-node)\n"
```

Comment:
This will be useful to Stytch developers. Do we think this is too confusing to include for external devs?

Reply:
we usually haven't done this in the past, mostly because we don't know where the `stytch-node` package is installed locally relative to the example app. However, I'm happy to leave it in